doc/team_practices.md

```diff
@@ -0,0 +1,37 @@
+# C2 Team Practices
+
+This document outlines best practices for the C2 team, and anyone wanting to contribute to the project.
+
+## Workflow
+
+All code changes should originate in a story on the [story tracker](https://pivotaltracker.com/n/projects/1149728).
```

Comment:
Maybe another way to say it: if I need to set time aside for it, or it requires discrete testing/review, then it needs a Story. Otherwise it's a freebie, a 2-fer, with the originating Story. Either way, it is getting tracked.